dashboard/src/api/client/query.tsx

```diff
@@ -0,0 +1,56 @@
+import { QueryCache, QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { AxiosError } from 'axios';
+import { StatusCodes } from 'http-status-codes';
+import { FC, ReactNode } from 'react';
+import { toast } from 'react-toastify';
+
+export interface GeneralAPIResponse {
+    error?: {
```

Comment:
You are exporting this interface, so my guess is that this interface is being used multiple places. create a folder types and create a file index.tsx and add the global type there.

Reply:
No. It's being used only here. Deleted export

Reply:
😎